rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -25,6 +25,10 @@ while /bin/true; do
     && pkill -9 --ns 1
```

Comment:
I found this not always works especially the user process like PS never exists.

Reply:
Maybe we can try "kill -9 1" to kill the docker root process explicitly.

---
In reply to: [161952341](https://github.com/Microsoft/pai/pull/235#discussion_r161952341) [](ancestors = 161952341)

Reply:
fixed by adding a docker container liveness file and changing `source` to `/bin/bash`